### DISCUSSION TITLE
ESP32 bluetooth + TDA1541(or 8xTDA1387 replacement)

### DISCUSSION THREAD

**Discussion by OxF8** [pschatzmann/ESP32-A2DP#406] [Show and tell]
Finally I succeed to make bluetooth with acceptable DAC. Thank You Phil!
https://drive.google.com/file/d/1ZNu3OEjSz_v6dEPZGL-wlKrgiFsMjcof/view?usp=sharing

**Comment by rpv-tomsk** [comment]
Unfortunately, link expired.
